### PR TITLE
feat(mojaloop/#2928): upgrade logger to central-services-logger

### PR DESCRIPTION
**Pull request by kleyow**
feat(mojaloop/#2928): upgrade logger to central-services-logger - https://github.com/mojaloop/project/issues/2928

- Switch lib logger to central services logger
- Improve testing and coverage
- Update dep
- Switch tests to serial to squash conflict with sinon stubbing
- Reduce logger noise by setting health endpoint logs to debug

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -89,15 +91,44 @@ async function rewriteContentTypeHeader(ctx, next) {
     await setConfig(process.env);
     const conf = getConfig();
 
+
     // Set up a logger for each running server
-    const space = Number(process.env.LOG_INDENT);
-    const transports = await Promise.all([
-        Transports.consoleDir(),
-        Transports.sqlite(process.env.SQLITE_LOG_FILE),
```
Comment (kleyow) [inline src/index.js:96]: Is it important to log to sqlite? Do we do this anywhere else?

File: src/index.js [hunk 2]
```diff
@@ -89,15 +90,11 @@ async function rewriteContentTypeHeader(ctx, next) {
...
-    ]);
-    const simLogger = new Logger({ context: { app: 'simulator' }, space, transports });
-    const reportLogger = new Logger({ context: { app: 'report' }, space, transports });
-    const testApiLogger = new Logger({ context: { app: 'test-api' }, space, transports });
+    const simLogger = Logger;
```
Comment (mdebarros) [inline src/index.js]:
```suggestion
    const simLogger = Logger.child({
            context: {
                app: 'simulator'
            }
        });
```

File: src/index.js [hunk 3]
```diff
@@ -89,15 +90,11 @@ async function rewriteContentTypeHeader(ctx, next) {
...
+    const reportLogger =  Logger;
```
Comment (mdebarros) [inline src/index.js]:
```suggestion
    const reportLogger =  Logger.child({
            context: {
                app: 'report'
            }
        });
```

File: src/index.js [hunk 4]
```diff
@@ -89,15 +90,11 @@ async function rewriteContentTypeHeader(ctx, next) {
...
+    const testApiLogger =  Logger;
```
Comment (mdebarros) [inline src/index.js]:
```suggestion
    const testApiLogger =  Logger.child({
            context: {
                app: 'test-api'
            }
        });
```

**Review by mdebarros — CHANGES_REQUESTED** [review]
LGTM, just a minor suggestion change.

Let me know if you agree.

**Review by mdebarros — DISMISSED** [review]

**Review by elnyry-sam-k — APPROVED** [review]

**Pull request merged**